idea-valley/Kandidatas.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+class Kandidatas extends IdeaValley
```

Comment:
Extending Candidate from Company is the same as extending Person from Car, because it drives a car.

It is composistion (https://www.youtube.com/watch?v=dYUZiJEy0JE&feature=youtu.be):
- `IdeaValley` has several `Candidate`
- `Candidate` mistaken applying to `IdeaValley` N..1 :)